src/Atoms/Calander.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import TextField from '@material-ui/core/TextField';
+import { makeStyles } from "@material-ui/styles";
+
+const useStyles = makeStyles({
+    textfield: {
+        width: "300px",
+        marginTop: "10px",
+    },
+})
+
+const Calander = (props) => {
+    
+    const classes = useStyles();
+    var dateFormat = require('dateformat')
```

Comment:
require statements should go along with imports or just use ES6 syntax across the app and remove require